.travis.yml

```diff
@@ -4,7 +4,7 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
```

Comment:
We still have 0.7 in REQUIRE, so we should either bump the version there or keep it in the tests.

Reply:
Right, let's keep it in Travis then, since we're expecting it to work on 0.7.